src/sec_security_json_yajl.c

```diff
@@ -1150,6 +1150,7 @@ Sec_Result SecJson_GenClose(Sec_JsonGenCtx *ctx, char *result, SEC_SIZE max_len)
         goto cleanup;
     }
 
+    result = malloc(sizeof(yajl_buffer_len));
```

Comment:
This will create a memory leak.
I checked with Davor and it looks like the correct solution is to have the function return an error result if result is NULL.

I suggest removing this malloc.  And adding an:
else
  goto cleanup;
after the    if (result != NULL && max_len > 0)
call on line 1136.